example/controller-registration.yaml

```diff
@@ -4,11 +4,13 @@ kind: ControllerRegistration
 metadata:
   name: kupid
 spec:
+  resources:
+  - kind: ClusterPorSchedulingPolicy
+    type: 
   deployment:
     type: helm
-    policy: Always
```

Comment:
I think you need the `policy: Always`

Reply:
I actually didn't change the example. It was generated I think as part of `make revendor`.  A stupid question, do you know what could have caused this. I've only modified the main.go with the relevant changes.

Reply:
Fixed with [commit](https://github.com/gardener/kupid/pull/30/commits/b94cf3c5dc64dd9ae7c54c27398e6af274d61162)